pkg/selector/outputs/outputs.go

```diff
@@ -211,3 +194,94 @@ func reverse(s string) string {
 	}
 	return string(runes)
 }
+
+// getWideColumnsData returns the column data necessary for a wide output for each of
+// the given instance types
+func getWideColumnsData(instanceTypes []*instancetypes.Details) []*wideColumnsData {
+	columnsData := []*wideColumnsData{}
+
+	for _, instanceType := range instanceTypes {
+		none := "none"
+		hyperisor := instanceType.Hypervisor
+		if hyperisor == nil {
+			hyperisor = &none
+		}
+
+		cpuArchitectures := []string{}
+		for _, cpuArch := range instanceType.ProcessorInfo.SupportedArchitectures {
+			cpuArchitectures = append(cpuArchitectures, *cpuArch)
+		}
+
+		gpus := int64(0)
+		gpuMemory := int64(0)
+		gpuType := []string{}
+		if instanceType.GpuInfo != nil {
```

Comment:
nit: when `gpuInfo` is nil because we have no gpus, wdyt about about having the default text be "none", "n/a", or etc. to match what we do with other "not present" cases?

Reply:
I will look into the reason why there was previously no default text for missing `gpuInfo` and then I will address it in my next PR.